frontend/src/routes/Components/SecondaryButton.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+class SecondaryButton extends Component {
+  render() {
+    const secondary = {
+      backgroundColor: "#0364FF",
+      borderRadius: "0.625rem",
+      bottom:"0rem",
+      right:"15rem"
```

Comment:
what do these do? is this to center the text? or add padding? cuz i think i wouldve done "padding: 1rem" or something and "text-align: center"